### PR TITLE
Improve documentation for server metricset in Zookeeper metricbeat module

### LINKED ISSUE #10376: Improve documentation 'server' Metricset in Zookeeper Metricbeat module
`server` Metricset uses the `srvr` [4 letter word](https://zookeeper.apache.org/doc/r3.1.2/zookeeperAdmin.html#sc_zkCommands) to extract and parse data from a Zookeeper instance. 

[Our documentation](https://github.com/elastic/beats/blob/master/metricbeat/module/zookeeper/server/_meta/fields.yml) and [here](https://github.com/elastic/beats/blob/master/metricbeat/module/zookeeper/server/_meta/docs.asciidoc) of the fields returned by this _endpoint_ is not really detailed. It will be nice to improve it and describe a good description of each field

### PR DESCRIPTION
**Pull request by arsalanc-v2**
Resolve #10376 

Fields to improve:
- [ ] connections
- [ ] avg latency
- [ ] max latency
- [ ] min latency
- [ ] mode
- [ ] node_count
- [ ] outstanding
- [ ] received
- [ ] sent
- [ ] version_date
- [ ] zxid
- [ ] count
- [ ] epoch

### CODE REVIEW

**Comment by sayden** [comment]
Hi @arsalanc-v2 . An absolutely amazing job. Even I'm learning things of Zookeeper I didn't know with this descriptions.

You still need to sign the CLA, read https://github.com/elastic/beats/blob/master/CONTRIBUTING.md to see how. Also, a change in `fields.yml` requires an update of the docs a few things more. You can achieve this by running `make update` from Metricbeat folder.

**Comment by arsalanc-v2** [comment]
Thank you! I have signed the CLA, please let me know if there is anything else I need to do to make the check pass.

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -81,7 +81,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - The `elasticsearch/deprecation` fileset now indexes the `component` field under `elasticsearch` instead of `elasticsearch.server`. {pull}10445[10445]
 - Remove field `kafka.log.trace.full` from kafka.log fielset. {pull}10398[10398]
 - Change field `kafka.log.class` for kafka.log fileset from text to keyword. {pull}10398[10398]
-- Address add_kubernetes_metadata processor issue where old source field is 
+- Address add_kubernetes_metadata processor issue where old source field is
```
Comment (sayden) [inline CHANGELOG.next.asciidoc]: This shouldn't go here. It seems some branching conflict. Try to rebase to master again and running `make collect && make update`

**Comment by sayden** [comment]
Left you some comments, it seems you need a rebase ensure that you aren't adding anything unrelated to the branch. Then `make collect && make update`

**Comment by arsalanc-v2** [comment]
When running either `make collect` or `make update`, an error appears exactly as the one mentioned here #7489 . However, the solution proposed there (manually creating the virtual environment) still results in the same error. It occurs right after "Generated fields.yml for metricbeat to ..." 

It seems like a common problem with other projects but I have not found a solution that would be applicable here

**Comment by sayden** [comment]
Yeah, it's common because our build system is done with Python 2 and it's tricky to make it work if you also have Python 3 installed. Ensure that you have Python 2 installed, of course :)

The workaround there might work, however. Try this way:

```
VIRTUALENV_PARAMS="-p python2.7" make python-env
source build/python-env/bin/activate
VIRTUALENV_PARAMS="-p python2.7" make collect-docs
VIRTUALENV_PARAMS="-p python2.7" make update
```

To really force

**Comment by arsalanc-v2** [comment]
Unfortunately that does not work either. 
image

Is it possible that despite the initial Python 2.7 deprecation message, Python 3 is still being used for subsequent processes? Also, would you recommend I uninstall Python 3?

**Comment by ruflin** [comment]
@arsalanc-v2 It's just a guess but I have seen similar things in the past having both python version and some dependencies for Python 2.7 were taken from Python 3 and that created a mess. Uninstalling Python 3 could definitively help and then remove / install the dependencies again. But not sure if that is a bit too much to ask to change your env.

Seeing you are on OS X, do you use brew? Perhaps you could use `pyenv` to manage your python versions?

**Comment by sayden** [comment]
Yes, it's definitely using Python 3 in some moment. It has happened to me on the past too. 

You probably have a symlink from `python` command to your `python3` binary. You can also try to backup this symlink (I don't know where it is located in OSX but check with `whereis python`) and then symlink `python2` binary to `python`. I think this is how I solved it

**Comment by arsalanc-v2** [comment]
@ruflin @sayden Thank you for your help! The commands ran successfully on an ubuntu vm.  

Yes I use brew and `pyenv` seems like a great tool, I'll be using it from now on.

**Comment by sayden** [comment]
@arsalanc-v2 don't forget that you need to sign the CLA https://www.elastic.co/contributor-agreement/

The PR looks done or almost done. Let's see what Jenkins thinks :)

**Comment by arsalanc-v2** [comment]
Awesome. I signed the CLA on 12/02/2019 and received the pdf by email. Is there anything else I need to do to make the check pass?

**Comment by jsoriano** [comment]
@arsalanc-v2 please check if you signed the CLA with the same email address as the used for your commits, you can see in https://github.com/elastic/beats/pull/10692.patch the email in the commits.

**Comment by arsalanc-v2** [comment]
The same email was used but the name was different. I've resolved that and the patch file shows the correct name now (:

**Comment by sayden** [comment]
jenkins, test this please

**Comment by jsoriano** [comment]
@arsalanc-v2 sorry, there was some issue with CLA signing last week, could you try to sign it again? :crossed_fingers:

**Comment by arsalanc-v2** [comment]
Done!

**Comment by sayden** [comment]
Awesome @arsalanc-v2 Let's wait CI to go green before merging :)

**Comment by sayden** [comment]
Oh, @arsalanc-v2 I didn't realize until now. Forgive me but we don't add CHANGELOG entries to docs updates. Can you remove it so we can merge the PR :slightly_smiling_face: ?

**Comment by arsalanc-v2** [comment]
@sayden Ah I see, no worries I've removed it. I read that from the contributing guide

**Comment by arsalanc-v2** [comment]
Also, I mistakenly pushed some old code (resolved now)

**Comment by sayden** [comment]
jenkins, test this

**Comment by sayden** [comment]
Good work @arsalanc-v2 :+1:

**Comment by ruflin** [comment]
@sayden Should we backport this?

**Pull request merged**